extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -62,8 +62,12 @@ Adds support for AT Internet. More details for adding AT Internet support can be
 
 Type attribute value: `baiduanalytics`
 
+<!-- markdown-link-check-disable -->
+
 Adds support for Baidu Analytics. More details for adding Baidu Analytics support can be found at [tongji.baidu.com/](http://tongji.baidu.com/web/help/article?id=268&type=0).
```

Comment:
If it's a dead link, shouldn't we just remove it instead?

Reply:
For some reason the `gulp check-links` task is marking these links as dead, even thought they're not when you manually check them.

One thing that I noticed was that the links redirect. However, when I change the link in the file to the redirect link, the task still flags it as a dead link. 

I think we should just disable the check for these links, since it seems like we have precedence to do so for other links in this doc, and this is blocking for another PR. 

@rsimha Any ideas as to why these links (and their redirects) are being counted as dead? 

The error code returned for the links is `0`, which signals no server response (https://www.npmjs.com/package/markdown-link-check) ...

Reply:
Neither link is returning a straightforward 200. First one is doing multiple redirects, second is returning an error code.

From httpstatus.io:

![image](https://user-images.githubusercontent.com/26553114/105107739-ba247100-5a86-11eb-874d-8cdca2f1c6b1.png)

Reply:
Good to know! Thanks @rsimha